.github/workflows/build_debian_package.yml

```diff
@@ -116,13 +123,15 @@ jobs:
 
       - name: Install smoke tests dependencies
         working-directory: ./smoke-tests
-        run: npm ci
+        run: |
+          npm install
+          npm ci
```

Comment:
Why do you need to add `npm install`? If you've fixed the package-lock.json problem in dev it should not be necessary I think